pyClient/zethGRPC.py

```diff
@@ -154,21 +160,24 @@ def parseHexadecimalPointBaseGroup2Affine(point):
 def make_empty_message():
     return prover_pb2.EmptyMessage()
 
-def parseVerificationKey(vkObj):
+def parseBctv14VerificationKey(vkObj):
```

Comment:
PGHR13 and BCTV14 are used interchangeably to refer to the same SNARK. Might be worth sticking to the same name to avoid potential confusions

Reply:
Fixed